index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>portfolio mobile first</title>
+  </head>
+  <body>
+    <div class="container">
+      <nav>
+        <p id="name">Henry Ik</p>
+        <img src="images/Union.png" alt="menu" />
+      </nav>
+      <main>
+        <section class="name">Hey There. I'm Henry</section>
```

Comment:
- [ ] please ensure that The container of the whole headline  section is a `section` tag . It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.